pkg/cloudprovider/providers/gce/gce_loadbalancer_external_test.go

```diff
@@ -237,3 +243,259 @@ func TestDeleteAddressWithWrongTier(t *testing.T) {
 		})
 	}
 }
+
+const (
+	gceProjectId = "test-project"
+	gceRegion    = "us-central1"
+	zoneName     = "us-central1-b"
+	nodeName     = "test-node-1"
+	clusterName  = "Test Cluster Name"
+	clusterID    = "test-cluster-id"
+)
+
+var apiService = &v1.Service{
+	Spec: v1.ServiceSpec{
+		SessionAffinity: v1.ServiceAffinityClientIP,
+		Type:            v1.ServiceTypeClusterIP,
+		Ports:           []v1.ServicePort{{Protocol: v1.ProtocolTCP, Port: int32(123)}},
+	},
+}
+
+func fakeGCECloud() (*GCECloud, error) {
+	client, err := newOauthClient(nil)
```

Comment:
This builds a real oauth client with application default credentials in our unit tests.